cpp/generate.rb

```diff
@@ -480,8 +530,10 @@ def blockFinderFor( name:"", tag_as:"", start_pattern:nil, needs_semicolon: true
     # because it's embedded inside of other patterns
     cpp_grammar[:template_call_innards] = template_call = newPattern(
         tag_as: 'meta.template.call',
-        match: lookBehindToAvoid(/</).then(/</).lookAheadToAvoid(/</).zeroOrMoreOf(characters_in_template_call).then(/>/).maybe(@spaces),
-        includes: [:template_call_context]
+        # if we add readable-support for recursive regex, then we might be able to use /<((?>[^<>]+|(\g<#groupNumberOfThisGroup>))*)>/ 
+        # to match the characters in the middle of a template call
```

Comment:
We also would need to confirm that vscode-textmate supports recursive regex.

Reply:
yeah, and I'd only give it a 50% chance that it does.